renovate/renovate.cabal

```diff
@@ -1,21 +1,35 @@
--- Initial renovate.cabal generated by cabal init.  For further 
--- documentation, see http://haskell.org/cabal/users-guide/
-
+cabal-version:       2.2
 name:                renovate
 version:             0.1.0.0
 synopsis:            An interface for binary analysis and rewriting
--- description:         
-license:             BSD3
+description:         Provides a set of tools for analyzing and rewriting
+                     binaries for various CPU architectures, including:
+                     * x86_64 (Intel)
+                     * ppc (PowerPC)
+                     * aarch32 (ARM)
+                     * aarch64 (ARM)
```

Comment:
Actually not aarch64

Reply:
OK, I have another batch of cabal changes coming, I'll remove it in that set.